start.sh

```diff
@@ -6,32 +6,38 @@ data_directory="data"
 LD_LIBRARY_PATH=$(esy x sh -c 'echo $LD_LIBRARY_PATH')
 export LD_LIBRARY_PATH
 
+# Start sidecli
```

Comment:
What is the pupose of sidecli?
I guess it would be great to have some information about:
- What it is
- Why do we need it
- How to use it